DQM/SiStripMonitorTrack/interface/SiStripMonitorTrack.h

```diff
@@ -233,9 +233,12 @@ class SiStripMonitorTrack : public DQMEDAnalyzer {
   edm::ParameterSet Parameters;
   edm::InputTag Cluster_src_;
 
+  edm::EDGetTokenT<edm::DetSetVector<SiStripDigi> > digiToken_;
   edm::EDGetTokenT<edmNew::DetSetVector<SiStripCluster> > clusterToken_;
   edm::EDGetTokenT<reco::TrackCollection> trackToken_;
 
+  edm::Handle<edm::DetSetVector<SiStripDigi>> digiHandle_;
```

Comment:
Potentially yes. 
The reason it is currently implemented as a class member is that since the function where it is actually uses is rather deep in the various methods called by the analyze() method, so initializing it where it is used or passing it though the method calls would require quite a larger alterations to the code.